test/WebJobs.Script.Tests.Integration/Management/HostAssignmentContextTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Script.WebHost.Models;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Managment
+{
+    public class HostAssignmentContextTests
+    {
+        [Theory]
+        [InlineData(null, null, false)]
+        [InlineData("", "", false)]
+        [InlineData("secret", "", false)]
+        [InlineData("", "endpoint", false)]
+        [InlineData("secret", "endpoint", true)]
+        public void ComputesMSIStatusBasedOnSecretAndEndpoint(string msiSecret, string msiEndpoint, bool expectedMsiEnabled)
```

Comment:
nit: make test name match method you're testing (e.g. TryGetMSIEndpoint_ReturnsExectedResult)

Reply:
Fixed